src/FilterWPQuery.php

```diff
@@ -65,19 +70,18 @@ public static function getFilterPriority() : int
 		return static::$filterPriority;
 	}
 
-
 	/** @inheritdoc */
 	public static function getPosts() : array
 	{
-		//Create 4 mock posts with different titles
+		// Create 4 mock posts with different titles.
 		$mockPosts = [];
-		for ($i = 0; $i <= 3; $i++) {
-			$post = new \WP_Post((new \stdClass()));
-			$post->post_title = "Mock Post $i";
+		for ($postNum = 0; $postNum <= 3; $postNum++) {
```

Comment:
I don't know why I always use $i instead of a semantic variable for a for loop, but this is way more readable.

Reply:
I get it.  I see that naming pattern propagated throughout the internet.  

But the languages we use allow us to use human-readable naming strategies.  I advocate that we take advantage of higher-level languages to make our code _tell us_ what's going on.

1. Variables should be named to _tell us_ what value it represents.
2. Function should be named to _tell us_ what it will do (it's expected behavior) when we invoke it.  It's action-based and should start with a verb.
3. Class should be a singular noun, as it represents a single object.
and so on....

When I first started in this profession back in the mid-80's, my first machine was in boolean logic.  It was not human-readable, but rather memory addressed. Yuck.  I was thrilled when human-readable languages allowed us to be make our code declarative.